bundles/org.openhab.binding.weathercompany/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,518 @@
+				<description>This is the daytime weather forecast in 4 days</description>
+			</channel-group>
+			<channel-group id="forecastDay4Night" typeId="forecastDaypart">
+				<label>Night-time Weather Forecast Day 4</label>
+				<description>This is the night-time weather forecast in 4 days</description>
+			</channel-group>
+
+			<channel-group id="forecastDay5Day" typeId="forecastDaypart">
+				<label>Daytime Weather Forecast Day 5</label>
+				<description>This is the daytime weather forecast in 5 days</description>
+			</channel-group>
+			<channel-group id="forecastDay5Night" typeId="forecastDaypart">
+				<label>Night-time Weather Forecast Day 5</label>
+				<description>This is the night-time weather forecast in 5 days</description>
+			</channel-group>
+		</channel-groups>
+
+		<config-description-ref uri="thing-type:weathercompany:weather" />
+	</thing-type>
+
+	<channel-group-type id="pwsObservations">
+		<label>PWS Current Observations</label>
+		<description>Current observations from Personal Weather Station</description>
+		<channels>
+			<channel id="neighborhood" typeId="neighborhood" />
+			<channel id="observationTimeLocal" typeId="observationTimeLocal" />
+			<channel id="currentTemperature" typeId="temperature">
+				<label>Current Temperature</label>
+			</channel>
+			<channel id="currentTemperatureDewPoint" typeId="temperature">
+				<label>Dew Point Temperature</label>
+			</channel>
+			<channel id="currentTemperatureHeatIndex" typeId="temperature">
+				<label>Heat Index Temperature</label>
+			</channel>
+			<channel id="currentTemperatureWindChill" typeId="temperature">
+				<label>Wind Chill Temperature</label>
+			</channel>
+			<channel id="currentHumidity" typeId="relativeHumidity">
+				<label>Current Humidity</label>
+			</channel>
+			<channel id="currentPressure" typeId="pressure">
+				<label>Current Pressure</label>
+			</channel>
+			<channel id="currentPrecipitationRate" typeId="precipitationRate">
+				<label>Current Precipitation Rate</label>
+			</channel>
+			<channel id="currentPrecipitationTotal" typeId="precipitationTotal">
+				<label>Current Precipitation Total</label>
+			</channel>
+			<channel id="currentSolarRadiation" typeId="solarRadiation">
+				<label>Current Solar Radiation</label>
+			</channel>
+			<channel id="currentUv" typeId="uvIndex">
+				<label>Current UV Index</label>
+			</channel>
+			<channel id="currentWindDirection" typeId="windDirection">
+				<label>Current Wind Direction</label>
+			</channel>
+			<channel id="currentWindSpeed" typeId="windSpeed">
+				<label>Current Wind Speed</label>
+			</channel>
+			<channel id="currentWindSpeedGust" typeId="windSpeed">
+				<label>Current Wind Gust Speed</label>
+			</channel>
+			<channel id="stationId" typeId="stationId" />
+			<channel id="country" typeId="country" />
+			<channel id="latitude" typeId="latitude" />
+			<channel id="longitude" typeId="longitude" />
+			<channel id="elevation" typeId="elevation" />
```

Comment:
you should merge these into a single channel of type "Location"

Reply:
@kaikreuzer I looked into making this change. The elevation used in `PointType` is defined to be in SI units (meters) only. As `PointType` does not use UoM, I don't understand how elevation can properly be represented in systems that are configured to use Imperial units. Maybe I'm missing something, so hopefully you can give some guidance.

Reply:
You are right - when introducing UoM, the `PointType` was indeed completely forgotten 😲 .
So yes, this indeed won't work if users want to use the (so outdated 😛 ) imperial units.

As the altitude is an optional value: How about merely merging latitude&longitude into a PointType channel, while keeping elevation a separate one?

Reply:
Perfect! I was going to suggest the exact same thing.

I dunno about those countries (or should I say country) that are still in the dark ages... LOL

Reply:
Completed.